_posts/2017-01-23-upgrade-guide.md

```diff
@@ -45,6 +45,36 @@ Note that the Babel compiler is only supported officially in these environments:
 
 This was removed and instead you should specificly deside which plugins you want to activate.
 
+Before:
+
+```js
+babylon.parse(code, {
+  plugins: [ "*" ]
+})
+```
+
+While this is not recommended for performmance issues, you can get the old behavior using:
```

Comment:
It wasn't removed for performance issues (it was about breaking changes in versions for these experimental plugins)